schema/assets/video-frame.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Video Frame",
+    "description": "Asset representing a frame from a video at a specific timestamp.",
+    "allOf": [
+        {
+            "$ref": "#/$defs/assets/asset"
+        },
+        {
+            "$ref": "#/$defs/helpers/slottable-object"
+        },
+        {
+            "type": "object",
+            "properties": {
+                "ty": {
+                    "title": "Type",
+                    "description": "Asset type",
+                    "type": "integer",
+                    "const": 3
+                },
+                "vsid": {

```

Comment:
Naming bike-shed: normally `sid` refers for "slot ID", which I guess could be adopted here if we squint.  But since we're not dealing with real slots, that may be confusing.

How about `refId`, which is the normal way to reference assets?